server/routes/prisons/sessionTemplates/addSessionTemplateController.ts

```diff
@@ -142,17 +146,14 @@ export default class AddSessionTemplateController {
           }
           return true
         }),
-      body('openCapacity').trim().isInt().withMessage('Enter a number'),
-      body(['openCapacity', 'closedCapacity'])
-        .trim()
-        .custom((_value, { req }) => {
-          const { openCapacity, closedCapacity } = req.body
-          if (openCapacity === '0' && closedCapacity === '0') {
-            throw new Error('Enter a capacity for either open or closed')
-          }
-          return true
-        }),
-      body('closedCapacity').trim().isInt().withMessage('Enter a number'),
+      body(['openCapacity', 'closedCapacity']).trim().toInt().isInt().withMessage('Enter a number'),
```

Comment:
Change here is to:
* convert open/closed capacity to integers (`toInt()`) and then confirm they are (`isInt()`)
* __then__, check at least one of them is greater than zero

Reply:
Will have to be careful where we use toInt / isInt - tried to use on the dates fields but it returns the value as an int (so cuts the 0 from 05-01 for instance) but here it works well